.travis.yml

```
@@ -0,0 +1,14 @@
+language: minimal
+
+git:
+  depth: 1
+
+before_install:
+    - ./checkout-sdk.sh
+
```

Can we cache the install directories?

Not sure if these are the right folders, but something like:
```suggestion

cache:
  directories:
    - docker/.composer/cache
    - docker/.drush/
    
```